src/docfx/build/legacy/LegacyFileMap.cs

```diff
@@ -33,7 +42,8 @@ internal static class LegacyFileMap
                         var legacyOutputFilePathRelativeToSiteBasePath = document.ToLegacyOutputPathRelativeToSiteBasePath(docset, publishItem);
                         var legacySiteUrlRelativeToSiteBasePath = document.ToLegacySiteUrlRelativeToSiteBasePath(docset);
 
-                        var fileItem = LegacyFileMapItem.Instance(legacyOutputFilePathRelativeToSiteBasePath, legacySiteUrlRelativeToSiteBasePath, document.ContentType);
+                        var monikerRange = GetMonikerRangeConfig(document);
```

Comment:
can we remove this? i am assuming that we can also write moniker range in file metadata

Reply:
i mean markdown yaml header

Reply:
The monikerRange used here is just a idenetity of group, it won't be used to be  evaluated to monikers.

---
In reply to: [293207261](https://github.com/dotnet/docfx/pull/4722#discussion_r293207261) [](ancestors = 293207261)